lib/galaxy/model/__init__.py

```diff
@@ -1510,7 +1510,13 @@ def __init__( self, history, action, role ):
         self.role = role
 
 
-class Dataset( object ):
+class StorableObject( object ):
+
+    def __init__( self, id, **kwargs):
+        self.id = id
```

Comment:
Am I interpreting https://github.com/mr-c/galaxy/blob/storableobject/lib/galaxy/model/__init__.py#L3698 wrong? Looks like `MetadataFile.id` is used.